### ISSUE TITLE
Refreshing browser page causes Filters dialog to lose options

### ISSUE THREAD

**Issue by LeoBreebaart** [radiantearth/stac-browser#490]
If you browse to a collection that allows spaciotemporal item filtering, and you refresh that page in the browser, everything except "Items per page" disappears from the 'Filters' dialog pane.

I have verified this behaviour for different STAC servers, and in different browsers (Firefox, Chrome, Safari on macOS, Firefox on Windows).

_Examples_: 
- [https://radiantearth.github.io/stac-browser/#/external/pgstac.demo.cloudferro.com/collections/landsat-c2-l1-oli](https://radiantearth.github.io/stac-browser/#/external/pgstac.demo.cloudferro.com/collections/landsat-c2-l1-oli)
- [https://radiantearth.github.io/stac-browser/#/external/emc.spacebel.be/collections/GOCE_Level_2](https://radiantearth.github.io/stac-browser/#/external/emc.spacebel.be/collections/GOCE_Level_2)

_Before & After Screenshots_:

![image](https://github.com/user-attachments/assets/42214af2-2e3d-4932-b822-6073fcc4e8fa)

![image](https://github.com/user-attachments/assets/aa043209-e769-4904-939d-37c73f16e104)

**Comment by m-mohr** [comment]
Yes, that's a limitation and will likely not be adressed anytime soon. If you refresh a page you have to expect that data gets lost, that's not only an issue in STAC Browser though. Why would you want to do that anyway?

**Comment by LeoBreebaart** [comment]
Yes, I can confirm that I've seen this behaviour before that commit yesterday as well.

Not sure if it helps, but as far as I can tell the only way to re-set the state so that the filter shows again, is to go 'Up' and then re-enter the collection. That way, the 'queryables' endpoint gets called again, which it doesn't if you just refresh the page, or copy and paste the url in a new tab or window -- not even a private window.

Regards,

-- Leo

**Comment by m-mohr** [comment]
Thanks for the information, I'll have a look...

**Comment by drnextgis** [comment]
I was about to open a ticket for the same issue, but I came across this one. I can confirm that the problem is easily reproducible.

**Comment by drnextgis** [comment]
Also as you can see from the screenshot, the 'Search' button disappears as well.

**Comment by m-mohr** [comment]
I've committed a fix in b5ebe6b8.
This may have side effects as I'm changing things at the heart of STAC Browser here, so we need to carefully test.
Could you check whether everything is still working for you and additionally that the bug is fixed? Thanks! @drnextgis @LeoBreebaart

**Comment by LeoBreebaart** [comment]
I can confirm that for the Spacebel collections and my own private STAC stuff, the problem now seems to be solved. The other example link I posted no longer works, but that seems to be an unrelated connection error, and I realise in hindsight that that may also have been a link I should not have publicly posted, so I removed it from my original post.

So please forget about that. Every other example I have now works fine, as far as I can tell. Many thanks for your quick response, Matthias -- we really appreciate it. (As we also do, indeed, appreciate the radiantbrowser itself, and your work on it. Thanks!) 

Regards,

-- Leo

**Comment by drnextgis** [comment]
I can confirm that the issue is resolved. Thank you!
